--- SIMVoleon-2.0.1.orig/lib/VolumeViz/misc/CvrCLUT.h	2004-12-20 15:54:48.000000000 +0100
+++ SIMVoleon-2.0.1/lib/VolumeViz/misc/CvrCLUT.h	2006-01-16 07:27:49.000000000 +0100
@@ -72,7 +72,7 @@
   void regenerateGLColorData(void);
 
   struct GLContextStorage {
-    GLContextStorage::GLContextStorage(uint32_t id)
+    GLContextStorage(uint32_t id)
     {
       this->texture1Dclut = 0;
       this->ctxid = id;
@@ -83,7 +83,7 @@
   };
   SbList<struct GLContextStorage *> contextlist;
   struct GlobalGLContextStorage {
-    GlobalGLContextStorage::GlobalGLContextStorage(void)
+    GlobalGLContextStorage(void)
     {
       this->fragmentprogramid[0] = this->fragmentprogramid[1] = 0;
     }
--- SIMVoleon-2.0.1.orig/lib/VolumeViz/render/common/CvrTextureObject.h	2004-12-20 15:54:48.000000000 +0100
+++ SIMVoleon-2.0.1/lib/VolumeViz/render/common/CvrTextureObject.h	2006-01-16 07:43:01.000000000 +0100
@@ -106,7 +106,7 @@
 
   static CvrTextureObject * findInstanceMatch(const SoType t,
                                               const struct CvrTextureObject::EqualityComparison & cmp);
-  unsigned long CvrTextureObject::hashKey(void) const;
+  unsigned long hashKey(void) const;
   static unsigned long hashKey(const struct CvrTextureObject::EqualityComparison & cmp);
 };
 
